cypress/integration/signIn.spec.ts

```diff
@@ -0,0 +1,59 @@
+describe('Sign In Page', () => {
+  context('given valid user credential', () => {
+    beforeEach(() => {
+      cy.mockSignInResponse(200, 'signInResponse/valid_response.json')
+      cy.signIn(Cypress.env('CYPRESS_VALID_EMAIL'), Cypress.env('CYPRESS_VALID_PASSWORD'))
+    })
+
+    it('redirects to home page', () => {
+      cy.get('[data-test-id=formButton]').click()
```

Comment:
we have a package called `@testing-library/cypress` in our template. We can use it to get test id easily.

`cy.findByTestId(testId)`

as there can be multiple test id for a group of test we can create another file to get those test Ids from. such as - 

in cypress/integration/SignIn/selectors.ts
```
const formSelectors = () => {
  button: 'formButton'
}

export { formSelectors } 
```

and in cypress/integration/signIn.spec.ts

```
import { formSelectors } from './selectors'

`cy.findByTestId(formSelectors.button)`
```

This way maintaining those testIds become much easier as those might have the need to change.

Also, as there are two test files for signIn right now ( if you add the `selectors.ts` it will become three ) . How about moving them to a separate folder which might be named `cypress/integration/SignIn/`

Reply:
Sounds great! Fixed in c03e5264aab02980008b9b737a49067c8bdda334